src/main/java/com/mturko/bestshop/model/Product.java

```diff
@@ -1,21 +1,25 @@
 package com.mturko.bestshop.model;
 
-import org.springframework.data.annotation.Id;
-
+import javax.persistence.Id;
+import javax.persistence.Entity;
+import javax.persistence.Table;
 import java.time.LocalDateTime;
 
-public class Product {
+@Entity
+@Table(name = "product")
+ public class Product extends AbstractEntity{
+
     @Id
-    private String id;
+    private int id;
```

Comment:
You already have in place this property in the AbstractEntity class. It isn't required to define it again here